app/controllers/sessions_controller.rb

```diff
@@ -95,6 +105,8 @@ def do_login
 
   def login_failure(message)
     StatsD.increment "login.failure"
+    session[:mfa_expires_at] = nil
```

Comment:
This can be 🔥  since https://github.com/rubygems/rubygems.org/pull/3325/files#diff-cdec550eeeb8fc63be2b5687170f594651a8d0b7f0465c9d807baef392639b6eR63-R65 was added.